pages/foods/[id].js

```diff
@@ -38,8 +40,12 @@ export default function FoodDetailsPage({
     cat.food.dislikes.includes(id)
   );
 
-  function handleEdit() {
-    setIsEditing(true);
+  function handleEditGood() {
```

Comment:
Here, for example, you could turn the "handleEditGood" into a "handleEditGoodFood". That would perhaps be a bit more specific and you would know exactly what it belongs to. If you had to take over the changes, then of course you would have to change all the places where it is used.

Reply:
Thanks for your suggestions :) 

I have changed the naming of the function, I hope it's now better understandable :-)